.github/workflows/build_and_test_midstream_image.yml

```diff
@@ -31,45 +31,10 @@ jobs:
           podman --version
           cd Dockerfiles/midstream/
           podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0
-      - name: "Run example operator-metadata through midstream ci image  --> positive test"
-        continue-on-error: false
-        run: |
-          mkdir ./output_logs
-          podman run -it -v $PWD:/project/operator-test-playbooks -v ./output_logs:/project/output midstream_image:latest
-
-      - name: "Run example operator-metadata through midstream ci image  --> negative parsing test"
-        continue-on-error: true
-        run: |
-          rm -rf ./output_logs
-          mkdir ./output_logs
-          podman run -it -v $PWD:/project/operator-test-playbooks -v ./output_logs:/project/output -e IMAGE_TO_TEST=quay.io/cvpops/test-operator:missing-alm-examples-v1 midstream_image:latest
 
-      - name: "Succeed if the negative test fails"
+      - name: "Run unit tests"
         continue-on-error: false
-        if: ${{ failure() }}
-        run: |
-          exit 0
-
-      - name: "Fail if the previous job skips"
-        if: ${{ cancelled() }}
         run: |
-          exit 0
-
-      - name: "Run example operator-metadata through midstream ci image  --> negative operator-sdk validation test"
-        continue-on-error: true
-        run: |
-          rm -rf ./output_logs
           mkdir ./output_logs
-          podman run -it -v $PWD:/project/operator-test-playbooks -v ./output_logs:/project/output -e IMAGE_TO_TEST=quay.io/cvpops/test-operator:invalid-dependencies-v1 midstream_image:latest
-
-      - name: "Succeed if the negative test fails"
-        continue-on-error: false
-        if: ${{ failure() }}
-        run: |
-          exit 0
-
-      - name: "Fail if the previous job skips"
-        if: ${{ cancelled() }}
-        run: |
-          exit 0
+          podman run -it -v $PWD:/project/operator-test-playbooks -v ./output_logs:/project/output midstream_image:latest python3 /unit_tests.py
```

Comment:
@samvarankashyap  Why do we need to bind-mount playbooks into container image?
It means people in midstream will have to checkout playbooks as well. Which is not optimal solution.
They should have everything necessary in midstream image.

Reply:
@samvarankashyap  bump

Reply:
@lslebodn Since, we are currently checking out the code from PR. We would like to add the latest changes in PR to be mounted on to the container . The container would be able to access the latest changes in PR while running the tests.

Reply:
@samvarankashyap  Such use-case might work for pull-request testing.
But it will not work in production. Because we would delegate "checkout" of correct branch to midstream team.
And I do not remember such requirements from midstream team. They just want to use image and choose version of checks based on tag.

Reply:
@lslebodn This is useful for the building the latest and greatest. everytime when a PR is merged. 
However, There needs to be a step which builds on every new tag of operator-test-playbooks. 
Midstream image will have both latest and tagged versions of the images at anypoint. 
Latest being based on every latest commit. 
Does that make sense ? 
@jsztuka Do we have a github action that builds on every tag ?

Reply:
I'm not sure if on every tag, but on each pull request. I'm not sure if I follow, what do you mean by every tag?

Reply:
@jsztuka  Looks like its already there 
https://github.com/redhat-operator-ecosystem/operator-test-playbooks/blob/master/.github/workflows/build_and_push_image_on_tag.yml#L39 
building new image and pushing it on everytag 
we need to change it to the official image

Reply:
@samvarankashyap the problem is not with pushing image to quay.

The problem is with the building of the image. The current version of Dockerfile in PR  https://github.com/redhat-operator-ecosystem/operator-test-playbooks/blob/47bc1b952eacdb4c3ce4c9980f956a4f8450c985/Dockerfiles/midstream/Dockerfile
does not contain roles and playbooks inside the image. Which means that midstream people will have to pull image from quay using tag and also checkout playbooks with the same tag.

And Jiri did not change any logic in the Dockerfile. He just add unit test there. Please file a bug for that. We have to fix it ASAP but it should not block this PR

Reply:
Changed to the official image, can you please confirm this address.